pkg/features/kube_features.go

```diff
@@ -775,6 +775,12 @@ const (
 	// Enables support for the StorageVersionMigrator controller.
 	StorageVersionMigrator featuregate.Feature = "StorageVersionMigrator"
 
+	// owner: @chaunceyctx
+	// alpha: v1.31
```

Comment:
1.32

Reply:
And you should use https://github.com/kubernetes/kubernetes/issues/125031 versioned feature gate now.

Reply:
get it, and I will fix all test error :)